Apromore-Core-Components/Apromore-Portal/src/main/webapp/WEB-INF/zk.xml

```diff
@@ -400,15 +400,13 @@
             <listener-class>org.apromore.zk.ApromoreDesktopCleanup</listener-class>
     </listener>
 
-    <listener>
+   <!-- <listener>
             <listener-class>org.apromore.portal.ApromoreZKListener</listener-class>
-    </listener>
+    </listener> -->
 
-    <listener>
-            <listener-class>org.apromore.portal.LoggingZKListener</listener-class>
-    </listener>
```

Comment:
ApromoreZKListener had two functions: it listened for ZK Executions (servlet requests) to do the JWT checking (which is valid for you to remove here).  But it also listened for ZK WebApps (servlet contexts) to override the cookie configuration.  Removing the entire thing probably breaks the fix for correcting the cookie flags.  We could tweak Nginx to cover for this, but it's better if the app is correct on its own without Nginx.

The LoggingZKListener should not be removed.

Reply:
Done.